DependencyInjection/Compiler/SerializerConfigurationPass.php

```diff
@@ -16,6 +16,9 @@
 
 /**
  * Checks if a serializer is either set or can be auto-configured.
+ *
+ * @author Christian Flothmann <christian.flothmann@xabbuh.de>
```

Comment:
I think this is wrong. I didn't contribute anything to this class if I'm not mistaken. :)